apps/src/templates/sectionProgress/VirtualizedSummaryView.jsx

```diff
@@ -3,33 +3,55 @@ import { MultiGrid } from 'react-virtualized';
 import styleConstants from '../../styleConstants';
 import { sectionDataPropType, scriptDataPropType, studentLevelProgressPropType } from './sectionProgressRedux';
 import StudentProgressSummaryCell from '../sectionProgress/StudentProgressSummaryCell';
+import color from "../../util/color";
 
 // TODO(caleybrock): share these styles with detail view
 const styles = {
```

Comment:
couldn't figure out a reasonable way to use table constants because this setup is so different from what we've done before
https://github.com/code-dot-org/code-dot-org/blob/staging/apps/src/templates/tables/tableConstants.js#L7